packages/parser/src/middleware/parser.ts

```diff
@@ -0,0 +1,56 @@
+import { MiddyLikeRequest } from '@aws-lambda-powertools/commons/types';
```

Comment:
All these imports could be converted to `import type ...`, this way both TS and the reader know what they are.